website/src/categories/components/select/accessibility.md

```diff
@@ -7,32 +7,72 @@ eleventyNavigation:
 ---
 <section>
 
-## Accessibility content
+## Keyboard interactions
 
-This is the accessibility example content
+<div class="ds-table-wrapper">
+
+|Command|Description|
+|-|-|
+|Tab|When focus is outside the select, moves focus to the input container.|
+|Space/Enter|Launches/closes the select panel|
+|Up|When focus is inside the select panel, moves focus upwards through the option items in the select panel|
+|Down|When focus is inside the select panel, moves focus downwards through the option items in the select panel|
+|Home|When focus is inside the select panel, moves to the first available option item in the select panel|
+|End|When focus is inside the select panel, moves to the last available option item in the select panel|
```

Comment:
Maybe also mention (possibly in a note below the table) that the focus loops in the the panel, so when you are at the last option and you use arrow down it will select the first option.

Or maybe use the complete phrasing as in the checkbox docs:

|Arrow Keys | Once you are in a checkbox group you can navigate to the next checkbox by using the right or down arrow key. You can navigate back to the previous box with left or up. The focus indicator loops, so when you are at the last option and press "down" it will focus on the first option.|
|-- | --|